app/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -837,15 +838,62 @@ def test_data_fetch_in_loop(self, lifecycle, mock_delay):
         tasks.fetch_jetstream_data()
         mock_delay.assert_called_once_with(experiment.id)
 
+    @parameterized.expand(
+        [
+            (NimbusExperimentFactory.Lifecycles.LIVE_APPROVE_APPROVE,),
+        ]
+    )
+    @patch("experimenter.jetstream.tasks.fetch_experiment_data.delay")
+    def test_data_fetch_live(self, lifecycle, mock_delay):
+        # continue to fetch data while LIVE, even after proposed end date
```

Comment:
Oh your comment here seems to me like it should be the actual name of the test

Reply:
Smart!